MAINTAINERS

```diff
@@ -1,7 +1,6 @@
 Michael Crosby <michael@docker.com> (@crosbymichael)
 Alexander Morozov <lk4d4@docker.com> (@LK4D4)
-Rohit Jnagal <jnagal@google.com> (@rjnagal)
-Victor Marmol <vmarmol@google.com> (@vmarmol)
+Vishnu Kannan <vishnuk@google.com> (@vishnuk)
```

Comment:
Hmm, is this actually the right GitHub ID?  I expect it to be @vishh, not vishnuk ([who doesn't seem to have any public activity][1]).

[1]: https://github.com/Vishnuk